jeta/archive/operations.py

```diff
@@ -16,56 +15,68 @@
 
 from astropy.time import Time
 
-import jeta.archive.file_defs as file_defs
 from jeta.archive.utils import get_env_variable
 
 ENG_ARCHIVE = get_env_variable('ENG_ARCHIVE')
 TELEMETRY_ARCHIVE = get_env_variable('TELEMETRY_ARCHIVE')
 ALL_KNOWN_MSID_METAFILE = get_env_variable('ALL_KNOWN_MSID_METAFILE')
-# JETA_LOGS = get_env_variable('JETA_LOGS')
 
+DEFAULT_CONFIG_PATH = '/srv/jeta/jeta/config/parameters.hdf5'
+
+
+def get_default_config(config_path=DEFAULT_CONFIG_PATH) -> str:
```

Comment:
Should we require that the default config path is passed into the function? As it stands, it is defined here and in docker/jeta/install_config.py. Might be better to only have it defined in one place.